package.json

```diff
@@ -101,7 +100,6 @@
   "build": {
     "appId": "network.hathor.macos.wallet",
     "compression": "maximum",
-    "afterSign": "scripts/notarize.js",
```

Comment:
I feel we have to add `"notarize": "notarytool"` here. But that's not exactly what I tested.

Link: https://www.electron.build/configuration/mac.html

Reply:
Reading about the parameters to be passed to the build options, it seems that we can use all the default settings for notarization on our app.

I updated our pack settings to use them on [ef0cda4](https://github.com/HathorNetwork/hathor-wallet/pull/421/commits/ef0cda42a8662e0d9e98b0f3ada018f4b39f445e).

Reference: https://github.com/electron/notarize#method-notarizeopts-promisevoid

Reply:
Note: You MUST specify APPLE_ID and APPLE_APP_SPECIFIC_PASSWORD via environment variables to activate notarization step

Reply:
Added this description to the release docs on [b7f6b74](https://github.com/HathorNetwork/hathor-wallet/pull/421/commits/b7f6b749b640ba6c27993bbef0e21b20a63a6bc2).